apps/eas-expo-go/eas.json

```diff
@@ -42,7 +42,7 @@
         "EAS_BUILD_PROFILE": "versioned-client"
       },
       "android": {
```

Comment:
We need to modify the config, but I'll create a separate PR. I need to understand all the config versions and their usage before.

Reply:
Yeah, let's keep versioned/unversioned flavors in EAS and Fastlane for now, we'll address this separately.